app/models/user.rb

```diff
@@ -25,6 +25,7 @@ class User < ActiveRecord::Base
             format: { with: EMAIL_REGEX },
             if: :email
 
+  scope :pending_cards, -> { where('cards.review_date <= ?', Time.zone.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.